kvm-install-vm

```diff
@@ -298,8 +298,20 @@ function fetch_images ()
     then
         output "Cloud image not found.  Downloading"
```

Comment:
Could we move this output statement to the `else` block.  And if `*.part` exists, output a different message, for e.g.:

```
output "Partial cloud image found.  Resuming download"
```

Reply:
Good Idea. Will do.